package.json

```diff
@@ -29,7 +29,7 @@
 		"@kobalte/core": "^0.9.6",
 		"@typescript-eslint/eslint-plugin": "^5.59.9",
 		"@typescript-eslint/parser": "^5.59.9",
-		"astro": "^2.6.2",
+		"astro": "^2.9.7",
```

Comment:
Note that the `pnpm-lock.yaml` file in this PR contains a bunch of unrelated changes from me running [`pnpm dedupe`](https://pnpm.io/cli/dedupe) and updating Astro to the latest version.